db/migrate/20191023195941_assign_provider_name_to_contacts.rb

```diff
@@ -0,0 +1,8 @@
+class AssignProviderNameToContacts < ActiveRecord::Migration[6.0]
+  def change
+    ucas_contacts_without_name = Contact.where(name: nil)
```

Comment:
We should probably add `.includes(:provider)` here for performance.

Reply:
Before: `1.0257s` 
After: `0.7702s`

💯

Reply:
w00t! 😝 Ok, thought it would be a bit slower.